envpool/atari/atari_env.h

```diff
@@ -124,40 +131,67 @@ class AtariEnv : public Env<AtariEnvSpec> {
     }
     // init buf
     for (int i = 0; i < 2; ++i) {
-      maxpool_buf_.push_back(std::move(Array(raw_spec_)));
+      maxpool_buf_[i].reset(new uint8_t[kRawSize]);
```

Comment:
why not reuse the memory with std::move?

Reply:
Because we have to perform at least one memcpy, so moving the reference seems wasteful.

Reply:
Oh, sorry, this is the init function, there's no problem